packages/plugin-browser-device/device.js

```diff
@@ -1,10 +1,9 @@
-const cuid = require('@bugsnag/cuid')
 const assign = require('@bugsnag/core/lib/es-utils/assign')
 const BUGSNAG_ANONYMOUS_ID_KEY = 'bugsnag-anonymous-id'
 
-const getDeviceId = () => {
+const getDeviceId = (win = window) => {
```

Comment:
This doesn't need a default since it will always be passed `win` from the main plugin function

```suggestion
const getDeviceId = (win) => {
```